dashboard-for-web/articles/web-dashboard-designer-mode/appearance-customization/conditional-formatting.md

```diff
@@ -75,37 +67,49 @@ Select a condition from the list and [specify its settings](#appearance-settings
 	
 Specify additional settings in the **Miscellaneous** section. For example, you can specify the intersection level for the Pivot or apply the current rule to a row in the Grid.
 
-See the following topics for more information about specific format settings for dashboard items:
-
-* [Conditional Formatting - Grid](../designing-dashboard-items/grid/conditional-formatting.md)
-* [Conditional Formatting - Pivot](../designing-dashboard-items/pivot/conditional-formatting.md)
-* [Conditional Formatting - Card](../designing-dashboard-items/cards/conditional-formatting.md)
-
 ## Edit a Format Rule
 
-You can see existing format rules for an entire dashboard item. To do this, open the dashboard item's [Options](../ui-elements/dashboard-item-menu.md) menu and go to the **Conditional Formatting** section.
-
-![wdd-cf-all-rules](../../../images/img126046.png)
-
 To edit a format rule, select the rule and click **Edit** (the ![wdd-icon-edit-collection-value-item](../../../images/img126050.png) icon).
 
+![wdd-icon-delete-big](../../../images/wdd-grid-cf-edit-rule126025.png)
+
 Click **Delete** (the ![wdd-icon-delete-big](../../../images/img126104.png) icon) to delete the selected format rule.
 
 When you edit a format rule, you can enable or disable the rule in the **Miscellaneous** section.
 
-![](../../../images/web-conditional-formatting-edit-rule-miscellaneous-section.png)
+![miscellaneous-section](../../../images/web-conditional-formatting-edit-rule-miscellaneous-section.png)
+
+Other settings in the **Miscellaneous** section depend on the selected dashboard item.
 
 ## Appearance Settings
 
-The format rule menu's **Condition** section contains appearance settings. You can configure and customize the following format condition appearance settings:
+The format rule menu's **Condition** section contains appearance settings. 
+
+For Grid, Pivot, and Card items, you can configure and customize the current format condition appearance settings:
 
 * Choose a predefined background color or font in the **Appearance** tab.
 	
 	![wdd-cf-appearance-gallery](../../../images/img126044.png)
+
 * Add a predefined icon in the **Icons** tab.
-	
+
 	![wdd-cf-icons-gallery](../../../images/img126045.png)
 
-You can also customize predefined colors and values for each Range format rule.
 
-![wdd-cf-range-gallery](../../../images/img126043.png)
+Chart and Scatter Chart items have a predefined palette and a custom palette. Click a color chip in the **Custom Color** palette to set a new custom color. You can pick any color using the RGB or HSB color model in the invoked color picker.
+
+![wdd-cf-chart-styles](../../../images/wdd-cf-chart-styles.png)
+	
+You can change generated colors for the Range format rules:
+
+![wdd-cf-range-gallery](../../../images/wdd-cf-range-gallery126043.png)
+
+## Dashboard Items Specific
```

Comment:
Can we make this section title more specific? For example:

`## Dashboard Item Format Setting Specifics`

Reply:
Thank you. Please apply this suggestion.